src/components/Modal/Modal.tsx

```diff
@@ -36,7 +38,9 @@ function Modal(props: TModalProps) {
             <div className={scss.modalHeader}>
               <h2>{props.title}</h2>
               <div className={scss.headerButtons}>
-                <button onClick={props.onClickOutside} />
+                <button onClick={props.onClickOutside} type="button">
```

Comment:
```suggestion
                <button onClick={props.onClickOutside}>
```
esse type="button" não é necessário se você usa o html button ele só serve pra fazer atrocidades tipo usar div como button

Reply:
O pessoal estava aqui na call na hora que eu estava lendo e disseram que tem mais funcoes nisso, e disseram que e bom deixar.